Ghidra/Features/Base/src/main/java/ghidra/app/util/datatype/DataTypeUrl.java

```diff
@@ -26,7 +26,7 @@
 /**
  * A class to produce and parse URLs of the form:
  * <pre>
- * 	datatype:/12345678?uid=12345678&name=Bob
+ * 	datatype:/12345678?uid=12345678&amp;name=Bob
```

Comment:
Maybe @literal

Reply:
This text is inside of a **pre** tag, which means it does not need to be escaped.

Reply:
> This text is inside of a **pre** tag, which means it does not need to be escaped.

Even though it's within a **pre** tag, javadoc still generates an error for it. I'm going through a bunch of these and changing them to literal or code where more appropriate. From what I'm told @code implicitly adds the pre tag so the tags will be removed.